lom/src/plugins/plugins_files/sonic/plugin_integration_tests/linkcrc_mocker/linkcrc_utils/linkcrc_utils.go

```diff
@@ -51,6 +58,14 @@ func MockRedisWithLinkCrcCounters(pattern string, mockTimeInMinutes int, interfa
         Password: redis_password,
         DB:       redis_app_db,
     })
+
+    initialInterfaceStatuses, err := SaveInterfaceStatuses(appDbClient, mockedLinks)
```

Comment:
Can you add comments on why?

Reply:
This is to restore it later as present in restore function. We save and restore so that the links are bought back to original state.